src/Compilers/CSharp/Test/Semantic/FlowAnalysis/RegionAnalysisTests.cs

```diff
@@ -5171,7 +5190,7 @@ static void Main()
         /*</bind>*/
     }
 }");
-            Assert.False(results.Succeeded);
+            Assert.True(results.Succeeded);
```

Comment:
❓ Do we want to add assertions here for `results.Captured`?

Reply:
@sharwell This is going into a feature branch, I'm writing full tests up now. I just wanted to get the initial commit in.